src/ng-serializer.service.ts

```diff
@@ -1,6 +1,9 @@
 import { Injectable } from '@angular/core';
-import { Serializer } from '@kaiu/serializer';
+import { Serializer, Registry } from '@kaiu/serializer';
 
 @Injectable()
 export class NgSerializerService extends Serializer {
+    constructor(public readonly registry: Registry) {
```

Comment:
No need to make the registry public readonly in this constructor, the Parent constructor will take care of it

Reply:
You can inject the registry and then pass it to the parent constructor

```ts
constructor(registry: Registry) {
    super(registry);
}
```